Project/js/filter.js

```diff
@@ -0,0 +1,15 @@
+let list = document.getElementsByClassName("cities-list__element");
+
+function search() {
+    let input = document.getElementsByClassName("content-search__field")[0].value.toLowerCase();
+    for (i = 0; i < list.length; i++) {
+        if (list[i].getElementsByClassName("city-box__title")[0].innerHTML.toLowerCase().indexOf(input) > -1) {
```

Comment:
again, I'd use `querySelector` instead of `getElementsByClassName("city-box__title")[0]`, but that's a minor issue